x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -50,7 +50,21 @@ type HostIsolationExceptionPaginatedContent = PaginatedContentProps<
   typeof ExceptionItem
 >;
 
-/* eslint-disable complexity */
+const getPaginationObject = ({
+  total,
+  perPage,
+  page,
+}: {
```

Comment:
You can set default values up in the params as well, that way you won't have to use the nullish coalescing 
 operators below.

```
const getPaginationObject = ({
  total = 0,
  perPage = MANAGEMENT_DEFAULT_PAGE_SIZE,
  page = 1,
}: {
  total?: number;
  perPage?: number;
  page?: number;
}) 
```

Reply:
@ashokaditya : https://github.com/elastic/kibana/pull/124167/commits/bf9766e2642c8b5b5ee91ce525f1c0db43c462f8